sdk/core/azure-core/src/http/curl/curl_connection_private.hpp

```diff
@@ -128,6 +128,11 @@ namespace Azure { namespace Core { namespace Http {
       // Get the socket that libcurl is using from handle. Will use this to wait while
       // reading/writing
       // into wire
+#if !defined(_MSC_VER) && LIBCURL_VERSION_MAJOR <= 7 && LIBCURL_VERSION_MINOR < 45
```

Comment:
Why is this condition subtly different than the one before, why is this < 45, while above it was < 44?

Reply:
This is because its a different feature with a different release availabilltiy. But I have fixed this version test as well.